regtests/run_spark_sql.sh

```diff
@@ -13,11 +13,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# -----------------------------------------------------------------------------
+# Purpose: Launch the Spark SQL shell to interact with Polaris.
+# -----------------------------------------------------------------------------
 #
+# Usage:
+#   Without arguments: Uses local filesystem.
```

Comment:
Unclear what "uses local filesystem" means in this context, we should probably describe what this means in the doc

Reply:
Added more context